cor_graphical/code/cor_data_preprocess.R

```diff
@@ -103,10 +103,12 @@ dat.long.cor.subset$Dich_RaceEthnic = with(dat.long.cor.subset,
 # add LLoD value to show in the plot
 dat.long.cor.subset$LLoD = log10(llods[dat.long.cor.subset$assay])
 
+# add ULoQ value
+dat.long.cor.subset$ULoQ = with(dat.long.cor.subset, ifelse(assay %in% c("bindSpike","bindRBD"), log10(19136250), Inf))
 
 # reset Delta29overB & Delta57overB for response call later using LLoD & ULoQ truncated data at Day 1, Day 29, Day 57
-dat.long.cor.subset$Delta29overB = dat.long.cor.subset$Day29 - dat.long.cor.subset$B
-dat.long.cor.subset$Delta57overB = dat.long.cor.subset$Day57 - dat.long.cor.subset$B
+dat.long.cor.subset$Delta29overB = apply(dat.long.cor.subset[,c("Day29","ULoQ")], 1, FUN=min) - dat.long.cor.subset$B
```

Comment:
I think this is a good candidate for [`pmin`](https://stat.ethz.ch/R-manual/R-patched/library/base/html/Extremes.html) rather than `apply`

Reply:
> I think this is a good candidate for [`pmin`](https://stat.ethz.ch/R-manual/R-patched/library/base/html/Extremes.html) rather than `apply`

I'll just go ahead and merge; mostly wanted to make you aware of this function @yiwenlu